sources/packages/backend/workflow/BPMN/camunda-8/assessment-gateway.bpmn

```diff
@@ -708,7 +708,7 @@
+          <zeebe:output source="={ &#10;  studentData: {&#10;    dependantStatus: studentDataDependantstatus,&#10;    relationshipStatus: studentDataRelationshipStatus,&#10;    livingWithParents: studentDataLivingWithParents,&#10;    numberOfParents: if is defined(studentDataNumberOfParents) then studentDataNumberOfParents&#10;    else null&#10;  },&#10;  calculatedData: {&#10;    parentalAssets: if is defined(calculatedDataParentalAssets)&#10;    then calculatedDataParentalAssets else null,&#10;    studentMaritalStatusCode: if is defined(calculatedDataStudentMaritalStatusCode) &#10;    then calculatedDataStudentMaritalStatusCode else null,&#10;    totalEligibleDependents: calculatedDataTotalEligibleDependants,&#10;    familySize: calculatedDataFamilySize,&#10;    parentalAssetContribution: if is defined(calculatedDataParentalAssetContribution) &#10;    then calculatedDataParentalAssetContribution else null,&#10;    parentalContribution: if is defined(calculatedDataParentalContribution) &#10;    then calculatedDataParentalContribution else null,&#10;    parentDiscretionaryIncome: if is defined(calculatedDataTotalDiscretionaryIncome) &#10;    then calculatedDataTotalDiscretionaryIncome else null ,&#10;    dependantTotalMSOLAllowance: if is defined(calculatedDataDependantTotalMSOLAllowance) &#10;    then calculatedDataDependantTotalMSOLAllowance else null,&#10;    studentMSOLAllowance: if is defined(calculatedDataStudentMSOLAllowance) &#10;    then calculatedDataStudentMSOLAllowance else null,&#10;    totalChildCareCost: if is defined(calculatedDataTotalChildCareCost) &#10;    then calculatedDataTotalChildCareCost else null,&#10;    totalNonEducationalCost: if is defined(calaulatedDataTotalNonEducationalCost) &#10;    then calaulatedDataTotalNonEducationalCost else null,&#10;    dependantChildQuantity: if is defined(calculatedDataDependantChildQuantity) &#10;    then calculatedDataDependantChildQuantity else null,&#10;    dependantChildInDaycareQuantity: if is defined(calculatedDataDependantChildInDaycareQuantity)&#10;    then calculatedDataDependantChildInDaycareQuantity else null,&#10;    dependantInfantQuantity: if is defined(calculatedDataDependantInfantQuantity)&#10;    then calculatedDataDependantInfantQuantity else null,&#10;    dependantDeclaredOnTaxesQuantity: if is defined(calculatedDataDependantDeclaredOnTaxesQuantity)&#10;    then calculatedDataDependantDeclaredOnTaxesQuantity else null,&#10;    dependantPostSecondaryQuantity: if is defined(calculatedDataDependantPostSecondaryQuantity)&#10;    then calculatedDataDependantPostSecondaryQuantity else null,&#10;    partnerStudentStudyWeeks: if is defined(calculatedDataPartnerStudentStudyWeeks)&#10;    then calculatedDataPartnerStudentStudyWeeks else null,&#10;    pdppdStatus: calculatedDataPDPPDStatus&#10;  }&#10;}" target="workflowData" />
```

Comment:
minor. for the variable names we follow acronyms in uppercase. I would suggest `pdPPDStatus`

Reply:
I can change it, i thought there are 2 acronyms here, so kept it all in lower case, as they append them. I guess this is the first scenario we are encountering like appending 2 acronym as one. What are other devs suggestions @andrepestana-aot @lewischen-aot

Reply:
I'm fine with the way it is.